technology-considerations/droid_vs_siegfried.md

```diff
@@ -0,0 +1,22 @@
+## Droid vs Siegfried
+
+We need software to carry out the file format checks on the files uploaded to TDR. There are two choices, [Siegfried](https://github.com/richardlehane/siegfried) and [Droid](https://github.com/digital-preservation/droid)
+
+### Siegfried
```

Comment:
Could you add a note to say that we used Siegfried in the Alpha prototype because it wasn't possible to run DROID in a Lambda back then, but that it's now possible because of the new EFS support? It's the kind of detail that's easy to forget in a year's time.

Reply:
Yep, I've put that in.